app/common/state/tabContentState/partitionState.js

```diff
@@ -28,9 +22,9 @@ module.exports.getPartitionNumber = (state, frameKey) => {
 
   const partitionNumber = frame.get('partitionNumber')
   if (typeof partitionNumber === 'string') {
```

Comment:
@cezaraugusto do you know in what circumstances the partitionNumber may be a string? i want to make sure these are handled correctly now that private tabs are assigned a negative partitionNumber

Reply:
it used to be a string when you open a new tab (non-partitioned) from a partition but couldn't see it anymore. For context it still happens this way to [frames for example](https://github.com/brave/browser-laptop/blob/master/test/tab-components/tabTest.js#L84) for example but couldn't see it anymore in tabs.